views/main/index.ejs

```diff
@@ -0,0 +1,6 @@
+<h1>Link Shortner</h1>
+
+<form action="/links/" method="POST">
+  <input class="form-control" type="text" name="url" placeholder="Enter URL"  >
```

Comment:
change the input type from text to url, this will only accept a full address including the http:// and get around the redirect while fails unless it has the http:// attached to the address.